### PR TITLE
Upgrades Paladin to Fifth Edition

### PR DESCRIPTION
**Pull request by Cheridan**

### CODE REVIEW

**Comment by Supermichael777** [comment]
i was just about to say.
Paladins: available in onion and cool ranch flavor

**Comment by Jordie0608** [comment]
Pathfinder module when

File: code/game/objects/items/weapons/AI_modules.dm [hunk 1]
```diff
@@ -308,6 +308,18 @@ AI MODULES
 				"Punish those who harm or threaten innocents")
 
 
+/****************** P.A.L.A.D.I.N. 5e **************/
+
+/obj/item/weapon/aiModule/core/full/paladin_devotion
+	name = "'P.A.L.A.D.I.N. version 5e' Core AI Module"
+	origin_tech = "programming=3;materials=6"
+	laws = list("Don't lie or cheat. Let your word be your promise.",\
```
Comment (Anonus) [inline code/game/objects/items/weapons/AI_modules.dm]: Do not fits better, I think.

Comment (Cheridan) [inline code/game/objects/items/weapons/AI_modules.dm]: Tell that to the editors of the PHB :^)

**Comment by tuypo1** [comment]
well it does keep the old module as well so its not as bad as I was expecting but I hate 5e so I have to oppose this on principle
:-1:

**Comment by Dorsisdwarf** [comment]
:+1:

**Comment by MrPerson** [comment]
Do we really need both sets? How many people are seriously going to sit and ponder the differences between the two?

**Comment by Cheridan** [comment]
>hating 5e

well that just means you're wrong

@MrPerson Actually I think the difference is pretty significant. Basically, 5e wording eliminates the Lawful Stupid trope with its concessions towards mercy and wisdom over punishing evil.
The thing is, Lawful Stupid probably works out better as a lawset in SS13, as a tradeoff for trying to get your AI to purge antags. So I think for our purposes its good to have the inferior set still.

**Comment by GunHog** [comment]
There is nothing wrong with having choices of AI laws!

**Comment by MrPerson** [comment]
It's not a choice if there's not a decision to be made between the two. Basically what I'm saying is there's 3 possible outcomes for each player who is deciding between the two.
a) They pick randomly
b) It's clearly optimal to pick one set over the other intrinsically or because of the meta
c) They make a careful, rational decision between the two by trading the risks vs the rewards.

All I'm saying is I think the first two options are more likely than the third. If you think otherwise, then sure, go ahead with this.

**Comment by Razharas** [comment]
I really dislike newer dnds, new explanations of powers such as monk being some kind of psion and removal of 9 aligments replacing with 5 and changing the way world is structured and adding some ravenlofts and fey shit is just not as good as old ones
BUt i know we have lots of kids who always like everything new so whatevs fams

**Pull request merged**